app/views/admin/republishing/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :page_title, "Republish documents" %>
+<% content_for :title, "Republish documents" %>
+<% content_for :title_margin_bottom, 6 %>
+
+<div class="govuk-grid-row">
+  <section class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">
+      Sometimes it may be necessary to republish content to the Publishing API. This will refresh the content on the website.
+    </p>
+
+    <p class="govuk-body">
+      For example, if we make an update to govspeak and a publishing application pre-renders that content prior to its submission to Publishing API, that would require us to re-render and save new HTML for content.
+    </p>
+
+    <p class="govuk-body">
+      The following actions will allow you to schedule the republishing of content that was originally published in this application.
+      Any linked editions will also be republished through dependency resolution.
+      Try to pick the document type most focused to the scope of what you need to republish to avoid unnecessary server load.
+    </p>
+
+    <%= render "govuk_publishing_components/components/table", {
+      head: [
+        {
+          text: "Document title",
+        },
+        {
+          text: "Action",
+        },
+      ],
+      rows: [
+        [
+          {
+            text: "Past prime ministers",
```

Comment:
Maybe this could link to [the page](https://www.gov.uk/government/history/past-prime-ministers) that's actually being republished, so it's very clear to the user which page is being updated?